app/locustio/jira/http_actions.py

```diff
@@ -132,7 +137,10 @@ def search_jql(locust):
     jql = random.choice(jira_dataset['jqls'])[0]
 
     r = locust.client.get(f'/issues/?jql={jql}', catch_response=True)
-    assert locust.atl_token in r.content.decode('utf-8'), f'Can not search by {jql}'
+    content = r.content.decode('utf-8')
+    if not (locust.atl_token in content):
+        logger.error(f'Can not search by {jql}')
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')